test/src/core/stdlib.js

```diff
@@ -552,6 +553,13 @@ describe('foam.String', function() {
     */}))
       .toEqual("\n        multiline\n        function\n    ");
 
+      // expect(foam.String.multiline(function(
```

Comment:
I would just delete this rather than comment out. We should get rid of foam.String.multiline().